### DISCUSSION TITLE
Modded Server REI failed to synchronize recipe data

### DISCUSSION THREAD

**Discussion by HelpIsNice** [shedaniel/RoughlyEnoughItems#1110] [Q & A]
Playing on a modded 1.19 fabric server, hosted by Shockbyte. List of mods we have are tech reborn, Minegate, farmers delight, iron chest, waystones and a few others. We are having issue where it show up in chat upon login that "REI: Server failed to synchronize recipe data with the client. Please check the console logs for errors. This breaks REI and vanilla crafting books." The console doesn't really say why it break, and it was definitely working once before while we were testing out the server and the mods before. But it stop working after we reset and restart the world. The GUI is still there but no crafting recipe is shown when click. Is there a way to fix this problem?

Edit: according to the logs, it's cause by tech reborn. but am not exactly sure what is causing it. here's the log:

01.09 11:53:28 [Server] Netty Epoll Server IO #23/ERROR Error receiving packet 103
01.09 11:53:28 [Server] INFO io.netty.handler.codec.EncoderException: String too big (was 73680 characters, max 32767)
01.09 11:53:28 [Server] INFO at net.minecraft.class_2540.method_10788(class_2540.java:617) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2540.method_10814(class_2540.java:612) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at reborncore.common.crafting.RebornRecipeType.write(RebornRecipeType.java:81) ~[RebornCore-5.3.2.jar:?]
01.09 11:53:28 [Server] INFO at reborncore.common.crafting.RebornRecipeType.method_8124(RebornRecipeType.java:40) ~[RebornCore-5.3.2.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2788.method_17816(class_2788.java:52) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2540.method_34062(class_2540.java:179) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2788.method_11052(class_2788.java:27) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2545.method_10838(class_2545.java:45) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2545.encode(class_2545.java:14) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[netty-codec-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:808) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:306) ~[netty-transport-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2535.method_36942(class_2535.java:213) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at net.minecraft.class_2535.method_10761(class_2535.java:205) ~[server-intermediary.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) [netty-common-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469) [netty-common-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:391) [netty-transport-classes-epoll-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) [netty-common-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.76.Final.jar:?]
01.09 11:53:28 [Server] INFO at java.lang.Thread.run(Thread.java:833) [?:?]
01.09 11:53:28 [Server] Netty Epoll Server IO #23/INFO [STDOUT]: REI: Server failed to synchronize recipe data with the client! Please check the server console log for errors, this breaks REI and vanilla recipe books!
01.09 11:53:28 [Server] Server thread/INFO Replacing apoli:replaced_loot_table with null

**Comment by darksoulq** [comment]
Ill assume you have the "Every Compat" mod, if yes remove it, if not install either Extra Thicc Packets or Packet Fixer mod

**Reply by darksoulq** [reply]
Just realized i necroed this sorry
